src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/Dashboard.test.tsx

```diff
@@ -425,6 +425,33 @@ it("counts how often people click the 'Contact us' link", async () => {
   );
 });
 
+it("renders the dashboard with the action needed tab selected", () => {
+  const ComposedDashboard = composeStory(
+    DashboardUsNoPremiumNoScanNoBreaches,
+    Meta,
+  );
+  render(<ComposedDashboard activeTab="action-needed" />);
+
+  const tabActionNeededTrigger = screen.getByRole("tab", {
+    name: "Action needed",
+  });
+  expect(tabActionNeededTrigger.getAttribute("aria-selected")).toBe("true");
+});
+
+it("renders the dashboard with the fixed tab selected", () => {
```

Comment:
Nit, but the test descriptions contradict each other otherwise :)

```suggestion
it("renders the dashboard with the action needed tab selected when requested", () => {
  const ComposedDashboard = composeStory(
    DashboardUsNoPremiumNoScanNoBreaches,
    Meta,
  );
  render(<ComposedDashboard activeTab="action-needed" />);

  const tabActionNeededTrigger = screen.getByRole("tab", {
    name: "Action needed",
  });
  expect(tabActionNeededTrigger.getAttribute("aria-selected")).toBe("true");
});

it("renders the dashboard with the fixed tab selected when requested", () => {
```

Reply:
Updated in [4c80f52](https://github.com/mozilla/blurts-server/pull/4477/commits/4c80f52920d11d55fac806ad89bf098331c6c8ef).